internal/commands/login/inputs_test.go

```diff
@@ -184,10 +205,38 @@ func TestLoginInputs(t *testing.T) {
 
 			assert.Nil(t, tc.inputs.Resolve(profile, ui))
 
+			assert.Equal(t, tc.expectedURL, expectedURL)
+
 			console.Tty().Close() // flush the writers
 			<-doneCh              // wait for procedure to complete
 
 			tc.test(t, tc.inputs)
 		})
 	}
+
+	t.Run("should output error message if browser cannot be opened", func(t *testing.T) {
+		errMessage := "there was an issue opening your browser\n"
```

Comment:
[optional] also feel [meh] about needing to DRY this error message for assert purposes (also.. so formal with the newline at the end!)

Reply:
- will remove `errMessage` var
- new line is needed for assert :/ 

```
=== RUN   TestLoginInputs/should_output_error_message_if_browser_cannot_be_opened
    inputs_test.go:236: 
        failed to assert equals ( actual, expected )
        	"there was an issue opening your browser\n"
        	"there was an issue opening your browser"
--- FAIL: TestLoginInputs (0.00s)
    --- FAIL: TestLoginInputs/should_output_error_message_if_browser_cannot_be_opened (0.00s)
```

Reply:
yeah, the newline for the assert makes sense, but the error message in `errors.New` didnt need it, right?

Reply:
correct, I removed it